nall/GNUmakefile

```diff
@@ -139,6 +141,11 @@ ifeq ($(platform),bsd)
   options += -lstdc++ -lm
 endif
 
+# haiku settings
+ifeq ($(platform),haiku)
+  options += -lroot
```

Comment:
Out of curiosity, what does this do?

Reply:
In the Haiku world, libroot is basically the equivalent of the glibc in Linux. You'll find more details here : https://www.haiku-os.org/docs/api/group__libroot.html